komga-webui/src/components/ThumbnailExplorerDialog.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <v-dialog v-model="input" scrollable>
+    <v-card :max-height="$vuetify.breakpoint.height * .9" dark>
+      <v-card-text>
+        <v-container fluid>
+          <v-row class="mb-2 d-flex flex-sm-row flex-column align-center justify-space-around">
+
+            <v-pagination
+              v-model="page"
+              :total-visible="perPage"
+              :length="Math.ceil(thumbnails.length/perPage)"
+            ></v-pagination>
+
+            <div v-for="(url, i) in visibleThumbnails()"
+                 :key="i"
```

Comment:
```suggestion
                 :key="url"
```

`key` should be unique, since `i` is reused it may lead to issues on responsiveness.